### ISSUE TITLE
atari_py crashes when ALE setting color_averaging is set to true

### ISSUE THREAD

**Issue by hholst80** [openai/gym#335]
This works with the ALE but not with the one from atari_py

```
import atari_py
ale = atari_py.ALEInterface()
ale.setBool(b'color_averaging', True)
print('loading "pong.bin"')
ale.loadROM('pong.bin')
print('we will never see this line until the bug is fixed.')
```

**Comment by hholst80** [comment]
This was fixed in 0.2.11. I was using 0.2.6.

**Comment by hholst80** [comment]
My mistake. This still crashes.
